src/main/java/mate/academy/config/SecurityConfig.java

```diff
@@ -22,4 +29,23 @@ protected void configure(AuthenticationManagerBuilder auth) throws Exception {
         auth.userDetailsService(userDetailsService)
                 .passwordEncoder(passwordEncoder);
     }
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http
+                .httpBasic().disable()
+                .csrf().disable()
+                .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+                .and()
+                .authorizeRequests()
+                .antMatchers("/registration", "/login", "/inject").permitAll()
+                .antMatchers(HttpMethod.DELETE, "/products/**").hasRole("ADMIN")
+                .antMatchers(HttpMethod.DELETE, "/users/**").hasRole("ADMIN")
```

Comment:
You can replace this two line with just one that will be responsible for all Delete request to any endpoint .antMatchers(HttpMethod.Delete).hasRole("Admin" or Role.RoleName.ADMIN.name())